test/index.js

```diff
@@ -256,6 +257,11 @@ it('returns 401 handles when isValid false passed to validateFunc', (done) => {
     const request = { method: 'GET', url: '/always_reject', headers: { authorization: 'Bearer 12345678' } };
     server.inject(request, (res) => {
 
+        expect(res.result).to.equal({
+            statusCode: 401,
+            error: 'Unauthorized',
+            message: 'Missing authentication'
```

Comment:
I think this will need to be improved, we have supplied the token so we shouldn't receive a "missing authentication" error, we should get the "Bad token" error from before. If you implement the optional value above, I can take a look at the tests and these messages to make sure we don't create any breaking changes 👍